Two_Body_Guess/data/wtr_mbe_d_scf.py 

```diff
@@ -0,0 +1,7 @@
+{'Trimer-mbe_d_g': 6, 'Trimer-atom_g': 9, 
```

Comment:
I ran the water scripts without using `time_mbed.py`, so I have a separate file with scf cycles alone. I quickly realized I could save a lot more time if got a single function to give me both scf cycles and scf times from one execution.